textdb/textdb-dataflow/src/test/java/edu/uci/ics/textdb/dataflow/source/ScanBasedSourceOperatorTest.java

```diff
@@ -68,7 +69,8 @@ public void testFlow() throws DataFlowException, ParseException{
         int numTuples = 0;
         while((nextTuple  = scanBasedSourceOperator.getNextTuple()) != null){
             //Checking if the tuple retrieved is present in the samplesTuples
-            boolean contains = TestUtils.contains(tuples, nextTuple, TestConstants.ATTRIBUTES_PEOPLE);
+            boolean contains = TestUtils.contains(tuples, nextTuple, 
+                    Arrays.asList(TestConstants.ATTRIBUTES_PEOPLE));

```

Comment:
Why didn't we use the "containsAllResults()" helper function for this test case?